### PR TITLE
Fix lightbox caption and topbar to be in sync when toggling

### PR DESCRIPTION
**Pull request by muxin**
Fix https://github.com/ampproject/amphtml/issues/10391

### CODE REVIEW

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js [hunk 1]
```diff
@@ -31,6 +31,17 @@ import {numeric} from '../../../src/transition';
 const TAG = 'amp-lightbox-viewer';
 
 /**
+ * Set of namespaces that indicate the lightbox controls mode.
+ * Lightbox controls include top bar, description box
+ *
+ * @enum {number}
+ */
+const LIGHTBOX_CONTROLS_MODES = {
```
Comment (aghassemi) [inline extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: CamelCase for enum name itself.

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js [hunk 2]
```diff
@@ -229,12 +243,23 @@ export class AmpLightboxViewer extends AMP.BaseElement {
...
-  toggleDescriptionBox_() {
+  toggleDescriptionBox_(opt_display) {
     this.updateDescriptionBox_();
+    dev().assert(this.descriptionBox_);
+    if (opt_display == undefined) {
+      opt_display = this.descriptionBox_.classList.contains('hide');
+    }
     if (this.descriptionBox_.textContent) {
-      this.descriptionBox_.classList.toggle('hide');
+      if (opt_display) {
```
Comment (aghassemi) [inline extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: can be simplified to `this.descriptionBox_.classList.toggle('hide', opt_display);`

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js [hunk 3]
```diff
@@ -319,10 +344,19 @@ export class AmpLightboxViewer extends AMP.BaseElement {
...
    * @private
    */
-  toggleTopBar_() {
-    this.topBar_.classList.toggle('hide');
+  toggleTopBar_(opt_display) {
+    dev().assert(this.topBar_);
+    if (opt_display == undefined) {
+      opt_display = this.topBar_.classList.contains('hide');
+    }
+    if (opt_display) {
```
Comment (aghassemi) [inline extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: ditto

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js (hunk 2 above)
Comment (muxin) [reply extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: I found that IE doesn't support `classList.toggle` with second argument (see [this](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)), so we'd better use the dumb way.

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js (hunk 3 above)
Comment (muxin) [reply extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: same as above

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js (hunk 1 above)
Comment (muxin) [reply extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: done

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js (hunk 2 above)
Comment (aghassemi) [reply extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: we polyfill it https://github.com/ampproject/amphtml/pull/6400 and use it a lot actually.

Comment (muxin) [reply extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: done

File: extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js (hunk 3 above)
Comment (muxin) [reply extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js]: done

**Pull request merged**